e2e/index.js

```diff
@@ -8,5 +8,13 @@ const rootCommand = path.join(process.cwd(), 'bin/docsify');
 
 test('shows up help message without any args', async t => {
     const { stderr } = await execa(rootCommand, {reject: false});
-    t.snapshot(stderr)
-});
\ No newline at end of file
+    t.snapshot(stderr);
+});
+
+const matchSnapshot = async (t, arg) => {
+  const { stdout } = await execa(rootCommand, [arg]);
+  t.snapshot(stdout);
+};
+
+test('shows help with -h flag', matchSnapshot, '-h');
+test('shows help with --help flag', matchSnapshot, '--help');
```

Comment:
@jamesgeorge007 how the `args` will be passed in `matchSnapshot` ? it's not passing as params here.
i am thinking you are trying to achieve is that `matchSnapshot` will automatically take the next arg of `test` as its second args `arg` and 1st arg of `test` as its first args `(t)`?

Reply:
Exactly :+1:

Reply:
cool 👍